cloudprem/modules/replicated/replicated.json

```diff
@@ -0,0 +1,7 @@
+{
+  "DaemonAuthenticationType": "password",
+  "ReleaseSequence": ${release_sequence},
+  "DaemonAuthenticationPassword": "dozuki",
+  "TlsBootstrapType": "self-signed",
+  "TlsBootstrapHostname": "${nlb_hostname}"
```

Comment:
Hmm. For consistency here, i think we either want to quote this ReleaseSequence expansion, or _unquote_ the TlsBootStrapHostname? I think i'd prefer to see the latter...

Reply:
Can't do that because TlsBootStrapHostname NEEDS to be a string. ReleaseSequence NEEDS to be an integer. If you pass an integer in quotes replicated will ignore it 🙄

Reply:
Gotcha.

Reply:
JSON building specific.